### PR TITLE
Implemented missing transportation types

### PR DESCRIPTION
**Pull request by arnasbr**
driving+ferry, cycling+ferry, cycling+public_transport

**EDIT** driving+ferry and cycling+ferry were already present, only adding cycling+public_transport

### CODE REVIEW

File: traveltimepy/mapper.py [hunk 1]
```diff
@@ -258,16 +264,17 @@ def create_districts(
 
 
 def create_sectors(
-    coordinates: List[Coordinates],
```
Comment (danielnaumau) [inline traveltimepy/mapper.py:261]: don't change tabs

File: traveltimepy/mapper.py [hunk 2]
```diff
@@ -590,11 +602,11 @@ def create_routes(
 
 
 def create_proto_request(
-    origin: Coordinates,
-    destinations: List[Coordinates],
-    transportation: ProtoTransportation,
-    travel_time: int,
-    one_to_many: bool = True,
+        origin: Coordinates,
```
Comment (danielnaumau) [inline traveltimepy/mapper.py:605]: everywhere

File: traveltimepy/dto/transportation.py [hunk 3]
```diff
@@ -42,3 +42,23 @@ class PublicTransport(BaseModel):
     pt_change_delay: Optional[int] = None
     walking_time: Optional[int] = None
     max_changes: Optional[MaxChanges] = None
+
+
+class DrivingFerry(BaseModel):
```
Comment (danielnaumau) [inline traveltimepy/dto/transportation.py:47]: it was already implemented in Ferry. You just needed to add CyclingPublicTransport

Comment (arnasbr) [reply traveltimepy/dto/transportation.py]: removed CyclingFerry and DrivingFerry

**Review by danielnaumau — APPROVED** [review]

**Pull request merged**